src/pages/basic_data/app_admin/central_import/CentralImportPage.tsx

```diff
@@ -6,51 +6,39 @@ import { useTranslation } from 'react-i18next';
 import { useHistory } from 'react-router-dom';
 import {
   FetchImportCandidatesParams,
-  ImportCandidateOrderBy,
   ImportCandidatesSearchParam,
-  SortOrder,
   fetchImportCandidates,
 } from '../../../../api/searchApi';
 import { ErrorBoundary } from '../../../../components/ErrorBoundary';
 import { ListPagination } from '../../../../components/ListPagination';
 import { ListSkeleton } from '../../../../components/ListSkeleton';
 import { SearchForm } from '../../../../components/SearchForm';
 import { SortSelector } from '../../../../components/SortSelector';
-import { ImportCandidateStatus } from '../../../../types/importCandidate.types';
 import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
+import { useImportCandidatesParams } from '../../../../utils/hooks/useImportCandidatesParams';
 import { stringIncludesMathJax, typesetMathJax } from '../../../../utils/mathJaxHelpers';
-import { SearchParam } from '../../../../utils/searchHelpers';
-import { CandidateStatusFilter } from '../../BasicDataPage';
 import { CentralImportResultItem } from './CentralImportResultItem';
 
-interface CentralImportPageProps {
-  statusFilter: CandidateStatusFilter;
-  yearFilter: number;
-}
+const thisYear = new Date().getFullYear();
 
-export const CentralImportPage = ({ statusFilter, yearFilter }: CentralImportPageProps) => {
+export const CentralImportPage = () => {
   const { t } = useTranslation();
   const history = useHistory();
   const params = new URLSearchParams(history.location.search);
-  const resultsParam = params.get(SearchParam.Results);
-  const fromParam = params.get(SearchParam.From);
-  const rowsPerPage = (resultsParam && +resultsParam) || ROWS_PER_PAGE_OPTIONS[0];
-  const page = (fromParam && resultsParam && Math.floor(+fromParam / rowsPerPage)) || 0;
 
-  const selectedStatusFilter: ImportCandidateStatus = statusFilter.NOT_IMPORTED
-    ? 'NOT_IMPORTED'
-    : statusFilter.IMPORTED
-      ? 'IMPORTED'
-      : 'NOT_APPLICABLE';
+  const importCandidateParams = useImportCandidatesParams();
+
+  const sizeParam = importCandidateParams.sizeParam ?? ROWS_PER_PAGE_OPTIONS[0];
+  const fromParam = importCandidateParams.fromParam ?? 0;
 
   const importCandidateQueryParams: FetchImportCandidatesParams = {
-    query: params.get(ImportCandidatesSearchParam.Query),
-    publicationYear: yearFilter,
-    importStatus: selectedStatusFilter,
-    orderBy: (params.get(ImportCandidatesSearchParam.OrderBy) as ImportCandidateOrderBy | null) ?? 'createdDate',
-    sortOrder: (params.get(ImportCandidatesSearchParam.SortOrder) as SortOrder | null) ?? 'desc',
-    from: page * rowsPerPage,
-    size: rowsPerPage,
+    query: importCandidateParams.queryParam,
+    publicationYear: importCandidateParams.publicationYearParam ?? thisYear,
```

Comment:
Søk på år virker ikke helt som ønsket enda. Er i dialog med @StigNorland om det.
En quick-fix er å bruke `publicationYear=2024,2024` i stedet for bare `publicationYear=2024`. Om det ikke kommer noen snarlig løsning gjør jeg det før merge

Reply:
nvm, legger bare inn år med kommaseparering med en gang